dashboard/app/controllers/api/v1/peer_review_submissions_controller.rb

```diff
@@ -60,7 +61,15 @@ def index
 
     # This query gets matching fully-hydrated models in the correct order.
```

Comment:
lol "fully hydrated" wut??

Reply:
HA ten points to whoever knows what this means